### PR TITLE
configurable inlets pod server port

### PR DESCRIPTION
**Pull request by deitch**
Signed-off-by: Avi Deitcher <avi@deitcher.net>

**How to categorize this PR?**

/area dev-productivity
/kind enhancement

**What this PR does / why we need it**:

The `hack/hook-me.sh` script explicitly chooses to set the inlets pod listening port as 9443. However, some controllers may be configured when deployed to listen on other ports. Since the service syncs regularly (every 5 mins or so), any change will get overwritten, making it not work.

Rather than trying to force everything into the same ports and worrying about synchronizing and backports and such, this just gives an option (not required) to set the port. If no option is provided, it defaults to 9443, like today.

**Special notes for your reviewer**:

Discussed online with @timebertt 

**Release note**:

```feature developer
Enable runtime configuration of hook-me.sh inlets pod port
```

### CODE REVIEW

**Comment by ghost** [comment]
Thank you @deitch for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: hack/hook-me.sh [hunk 1]
```diff
@@ -266,6 +272,9 @@ if [[ "${BASH_SOURCE[0]}" = "$0" ]]; then
   webhookServerPort=${3:-}
   [[ -z $webhookServerPort ]] && echo "Please specify webhook server port" && exit 1
 
+  inletsServerPort=${4:-}
+  [[ -z $inletsServerPort ]] && echo "inlets-server not specified, using default port of 9443" && inletsServerPort=9443
```
Comment (timebertt) [inline hack/hook-me.sh]:
```suggestion
  [[ -z $inletsServerPort ]] && echo "inlets-server port not specified, using default port of 9443" && inletsServerPort=9443
```

**Review by timebertt — COMMENTED** [review]
Thank you!
One nit, otherwise looks good.

File: hack/hook-me.sh (hunk 1 above)
Comment (deitch) [reply hack/hook-me.sh]: Pushed out with change. Thanks for finding that.

**Review by timebertt — APPROVED** [review]
/lgtm
/ok-to-test

**Comment by deitch** [comment]
@timebertt is this good to go?

**Comment by timebertt** [comment]
Hmm, pipeline was not triggered unfortunately.
Doing it manually now

**Pull request merged**